src/client/index.js

```diff
@@ -1 +1,30 @@
 console.log('this is working!!')
+
+function submitUsername () {
+  var username = document.getElementById('username').value
```

Comment:
what about **let** instead of **var** . can avoid some unintentional bugs in long term :dancer: